judy-adam-lab/test/constructor-test.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+const expect = require('chai').expect;
+const fs = require('fs');
+const bitmapConstructor = require('../model/bitmap-constructor');
+
+describe('Bitmap Constructor Module', function() {
+  var result;
+  before('parse the buffer file', function(done){
+    fs.readFile(`${__dirname}/../../assets/bitmap.bmp`, (err, data) => {
+      result = bitmapConstructor(data);
+      done();
```

Comment:
You are essentially testing the Node module fs with this code. When testing, you want to make sure that you're only testing the functionality of your own code.